data/qyyjt

```diff
@@ -0,0 +1 @@
+qyyjt.cn
```

Comment:
The domain `qyyjt.cn` can not be resolved.

Reply:
www.qyyjt.cn